docs/src/main.css

```diff
@@ -30,7 +30,7 @@ code {
 }
 
 .grid-container {
-  background-color: #ddd;
+  background-color: white;
```

Comment:
ah, right, this is for the [grid aka layout preview](http://localhost:3000/components/detail/layout) so that the text shows on a light background. But looks like the other classes are affecting the color now, so we can remove the `background-color` and `color` properties here.